clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -17,8 +17,7 @@ using namespace llvm::opt;
 
 SYCLInstallationDetector::SYCLInstallationDetector(
     const Driver &D, const llvm::Triple &HostTriple,
-    const llvm::opt::ArgList &Args)
-    : D(D) {}
+    const llvm::opt::ArgList &Args) {}
```

Comment:
what happened with driver?

Reply:
It's not currently being used, the sanitizer builds flagged it.